app/views/confirmation/remove_user.html.erb

```diff
@@ -14,10 +14,7 @@
       What happens next?
     </h2>
     <p>
-      Reliability engineering will remove your users from the gds-users base account. They'll send you an email once the work is complete.
-    </p>
-    <p>
-      Contact the team on slack channel <a href="https://gds.slack.com/messages/CAD6NP598">#reliability-eng</a> for support.
+      Engineering Enablement will remove your users from the gds-users base account. They'll send you an email once the work is complete.
```

Comment:
I'd suggest we remove the reference to a team here and just say similar to what we're saying on other pages something like:

"Your request to remove users from the gds-users base account will be reviewed, and if accepted, your users will be removed. You will receive and email once this is done."